pkg/commands/bitfield.ts

```diff
@@ -0,0 +1,46 @@
+import { type Requester } from "../http";
+import { Command, type CommandOptions } from "./command";
+
+type SubcommandArgs<Rest extends unknown[] = []> = [
```

Comment:
We generally prefix generics with`T`, so, `Rest` should be `TRest`. Let's make that change.